vnep_approx/treewidth_model.py

```diff
@@ -1398,7 +1454,10 @@ class SeparationLP_OptDynVMP(object):
     def __init__(self,
                  scenario,
                  gurobi_settings=None,
-                 logger=None):
+                 logger=None,
+                 number_further_mappings_to_add=5,
+                 number_initial_mappings_to_compute=100,
+                 restrict_edge_mapping_to_fixed_paths=False):
```

Comment:
believe that these parameters were removed by me and should be loaded from the algorithms yaml file; will need to check that.